README.md

```diff
@@ -43,20 +43,38 @@ Start the docker container and wait for the migration to conclude:
 ```bash
 docker-compose up -d
 ```
-### Step 5: Run the Application
+### Step 5: Apply Database Migrations (Alembic)
+
+
+1.  **Initialize Alembic -** The below command will create a `migrations` directory with necessary configuration files:
```

Comment:
You need to initialize alembic your self i.e. create migrations directory and autogenerate your migrations
This folder must be part of the project alongside with your code.
 
When deploying user just need to run `alembic upgrade head`

Here is some links for you about how to use Alembic: 
- [Hands-on artile on how to use alembic](url)
- [Alembic Tutorial](url)